docs/content/en/blog/news/nonssa-vs-ssa.md

```diff
@@ -0,0 +1,89 @@
+---
+title: From legacy approach to server-side apply
+date: 2025-02-25
+---
+
+From version 5 of Java Operator SDK [server side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
+is a first-class feature and is used by default to update resources.
+As we will see, unfortunately (or fortunately), using it requires changes for your reconciler implementation.
+
+For this reason, we prepared a feature flag, which you can flip if you are not prepared to migrate yet:
+[`ConfigurationService.useSSAToPatchPrimaryResource`](https://github.com/operator-framework/java-operator-sdk/blob/main/operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java#L493)
+
+Setting this flag to false will make the operations done by `UpdateControl` using the former approach (not SSA).
+Similarly, the finalizer handling won't utilize SSA handling. 
+The plan is to keep this flag and allow the use of the former approach (non-SSA) also in future releases. 
```

Comment:
Maybe add that it will be properly deprecated at least one major or minor in advance so people can prepare?

Reply:
So I don't want to deprecate it, unless there is of course a major architectural change.